endpoints/openrtb2/auction_test.go

```diff
@@ -1020,9 +1019,11 @@ func TestStoredRequests(t *testing.T) {
 		hardcodedResponseIPValidator{response: true},
 	}
 
+	testStoreVideoAttr := []bool{true, true, false, false}
+
 	for i, requestData := range testStoredRequests {
-		newRequest, errList := deps.processStoredRequests(context.Background(), json.RawMessage(requestData))
-		if len(errList) != 0 {
+		newRequest, impExtInfoMap, errList := deps.processStoredRequests(context.Background(), json.RawMessage(requestData))
+		if assert.Empty(t, errList) {
```

Comment:
I think you by mistake added the `assert.Empty` in the wrong place. This should still be `if len(errList) != 0 {`. You probably wanna change to `assert.NotEmpty` on L1070

Reply:
oh ok, modified